mapit_it/templates/mapit/base.html

```diff
@@ -1,88 +1,4 @@
-{% load static from staticfiles %}
+{% extends "mapit/base.html" %}
```

Comment:
Drive-by: This won't work I'm afraid because it's the same template being overridden.
Put the description in a separate template and then override that?